django/cantusdb_project/main_app/tests/test_views.py

```diff
@@ -2772,46 +2774,55 @@ def setUp(self):
     def test_url_and_templates(self):
         source = make_fake_source()
 
-        response = self.client.get(reverse("chant-create", args=[source.id]))
-        self.assertEqual(response.status_code, 200)
-        self.assertTemplateUsed(response, "chant_create.html")
+        with patch("requests.get", mock_requests_get):
+            response_1 = self.client.get(reverse("chant-create", args=[source.id]))
+            self.assertEqual(response_1.status_code, 200)
+            self.assertTemplateUsed(response_1, "chant_create.html")
 
-        response = self.client.get(reverse("chant-create", args=[source.id + 100]))
-        self.assertEqual(response.status_code, 404)
-        self.assertTemplateUsed(response, "404.html")
+            response_2 = self.client.get(
+                reverse("chant-create", args=[source.id + 100])
+            )
+            self.assertEqual(response_2.status_code, 404)
+            self.assertTemplateUsed(response_2, "404.html")
 
     def test_create_chant(self):
         source = make_fake_source()
-        response = self.client.post(
-            reverse("chant-create", args=[source.id]),
-            {
-                "manuscript_full_text_std_spelling": "initial",
-                "folio": "001r",
-                "c_sequence": "1",
-            },
-        )
+        with patch("requests.get", mock_requests_get):
+            response = self.client.post(
```

Comment:
Does a post request to `chant-create` make a call using `requests.get`?

Reply:
oh. this is a good point.

Let me think this through aloud: when we send a post request, the response will be an indication that the post request was received, along with a redirection status code indicating the next page the client should load. But `self.client.post` will not load the next page itself.

So long as my reasoning is correct, this patch is not necessary. Let me remove it. Thanks!